lib/system/sysio.nim

```diff
@@ -37,6 +37,8 @@ proc fread(buf: pointer, size, n: int, f: File): int {.
 proc fseek(f: File, offset: clong, whence: int): int {.
   importc: "fseek", header: "<stdio.h>", tags: [].}
 proc ftell(f: File): int {.importc: "ftell", header: "<stdio.h>", tags: [].}
+proc ferror(f: File): int {.importc: "ferror", header: "<stdio.h>", tags: [].}
+proc feof(f: File): int {.importc: "feof", header: "<stdio.h>", tags: [].}
```

Comment:
ferror yes, see proc fileError(f: File). 
feof I wanted to use it, and it works in my case, but it breaks "jester", so I left the endOfFile implementation as is.